numpy/ma/core.py

```diff
@@ -3918,13 +3914,10 @@ def __repr__(self):
                 dtype=str(self.dtype)
             )
             is_structured = bool(self.dtype.names)
-            key = '{}_{}'.format(
-                'long' if is_long else 'short',
-                'flx' if is_structured else 'std'
-            )
+            key = f"{'long' if is_long else 'short'}_{'flx' if is_structured else 'std'}"
```

Comment:
OK. Code is a bit obfuscated to start with.

Reply:
I think this was easier to read without the f-strings, since then it could be line-wrapped

Reply:
Ok, i understood. I thought code in a single line would be more compact and f-string will give readability for programmers.

Reply:
The true fix would be to break up the code, but don't do that for this PR :)